docs/how-to/service-protection-setup.md

```diff
@@ -0,0 +1,74 @@
+# Installing Kuadrant Service Protection into an existing OCM Managed Cluster
+
+## Introduction
+This walkthrough will show you how to install and setup the Kuadrant Operator into an [OCM](https://open-cluster-management.io/) [Managed Cluster](https://open-cluster-management.io/concepts/managedcluster/).
+
+## Prerequisites
+* Access to an Open Cluster Management (>= 0.6.0) Managed Cluster, which has already been bootstrapped and registered with a hub cluster
```

Comment:
In the https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/docs/how-to/mgc-setup.md guide, there is OCM 0.11.0 required.

Reply:
good spot, fixed. thought I'd made these consistent in an earlier commit but I guess not